### PR TITLE
Update libreoffice.rdef.in with new Icon

### PR DESCRIPTION
**Pull request by roired**
Included new Icon matching Haiku HIG for icon design.
This Icon was made in Icon-O-Matic

### CODE REVIEW

File: app-office/libreoffice/additional-files/libreoffice.rdef.in [hunk 1]
```diff
@@ -30,13 +30,22 @@ resource file_types message {
...
+	$"08023342253B08022540334708022545334C0806365D36553A573E553E5D3C5C"
+	$"0A093E5D535353416045544B5B4D57506053495D170A000100000A010101000A"
+	$"020102000A030103000A040104000A050105000A06010630201E01178100040A"
+	$"07010730201E01178100040A08010830202101178100040A0901093020260117"
+	$"8100040A0A010A100217810004158100040A0B010B1001178100040A0C010C10"
+	$"01178100040A0D010D1001178100040A0E010E1001178100040A0F010F100117"
+	$"8100040A1001101001178100040A1101111001178100040A1201121001178100"
+	$"040A1301131001178100040A13011330202501178100040A140114000A150115"
+	$"00"	
+	};
```
Comment (fbrosson) [inline app-office/libreoffice/additional-files/libreoffice.rdef.in]: Cool! Thanks @roired for your PR! (I'll have a look at the new icon asap.)

Hmm, I think the preferred style for the closing `}` is without a tab:
```diff
- 	};
+};
```
So, shall we drop the tab in line 51?

**Comment by humdingerb** [comment]
What does everyone think of including the Icon-O-Matic files in the additional-files folder?
I'd very much appreciate that. It make having a look at the icon much easier and people would be able to improve it.

**Comment by fbrosson** [comment]
@humdingerb: It would be nice indeed.
BTW, it would be nice to also add a command-line interface to Icon-O-Matic that would allow to convert the source file into a text file with the `resource vector_icon { ... }` bloc like what we have in the rdef file.
But I guess we should open an issue for this.

@roired, your new icon looks ok, even-though the current one looks closer to what an average user would expect for LibreOffice. In other words, if I was shown the icon without the name of the file then I would probably not have guessed that it was for LibreOffice.

But since the Haiku HIG looks perfect in your new icon I believe it is an improvement.

**_To all:_** Shall we merge this, once the tab in line 51 is removed and `REVISION` bumped?

@roired: If you wish I can update your PR with the tiny changes described above.

File: app-office/libreoffice/additional-files/libreoffice.rdef.in (hunk 1 above)
Comment (roired) [reply app-office/libreoffice/additional-files/libreoffice.rdef.in]: Good day,
I think the tab is a mistype from the copy-paste due to this being my first attempt at a PR.
Regards,
RR

**Comment by scottmc** [comment]
> @humdingerb: It would be nice indeed.
> BTW, it would be nice to also add a command-line interface to Icon-O-Matic that would allow to convert the source file into a text file with the `resource vector_icon { ... }` bloc like what we have in the rdef file.
> But I guess we should open an issue for this.

There will likely be a GCI task to do this. ;)

**Comment by roired** [comment]
Good day, 
Apologies, this was my first attempt at a PR. And I am not allowed to upload any image. 

"something went really wrong and we can't process that file" 

is what I get, no matter what file format I try to upload.

@fbrosson: feel free to update the PR. I presume I copy-paste the tab, and didn't realize of it.

Regards,
RR

File: app-office/libreoffice/additional-files/libreoffice.rdef.in (hunk 1 above)
Comment (roired) [reply app-office/libreoffice/additional-files/libreoffice.rdef.in]: so sure, can drop that tab :)

**Comment by fbrosson** [comment]
Before I merge, does anyone wish to:
1) have a look at the new icon?
2) have its source added to the `additional-files/` directory?

For question 1, what I did was copy an app (`cp /system/apps/BabyBe baby-copy`) to a temporary directory, copy the `resource vector_icon` block to a `new-icon.rdef` file and then:
```bash
rc -o new-icon.rsrc new-icon.rdef
xres -o baby-copy new-icon.rsrc
```
There might be an easier way to see a vector icon but I did not investigate.

For question 2, I believe it is OK to add the source of the icon and not use it in the recipe. But I don't know if everyone is OK with this, so comments are welcome!

**Comment by fbrosson** [comment]
Thanks @roired for posting the image.
I'm adding a "needs testing and review" label to see if there is a consensus for the new icon.
(It's better to discus _before_ the merge rather than finding comments _after_ the merge :)

**Comment by fbrosson** [comment]
In case someone wants to merge this before me, please don't forget to choose "Squash and merge", get rid of the unneeded text in the commit message and, instead, write a nice one-line commit message ;)

**Comment by scottmc** [comment]
Looks good to me.  How much do we lose by just having the .rdef info and not the IOM file?

**Comment by fbrosson** [comment]
Looking at the other 13 IOM files we already have in the repo, and which are between 5 KiB and 26 KiB, I'd say that this one is probably around 20 KiB.
@roired, is my guess correct ?

**Comment by roired** [comment]
> Looking at the other 13 IOM files we already have in the repo, and which are between 5 KiB and 26 KiB, I'd say that this one is probably around 20 KiB.
> @roired, is my guess correct ?
It's about 29,5kB. That is for the IOM file, but I can't upload it as is not a supported file format. If you point me where to send it, I will.
Regards, 
RR

**Comment by roired** [comment]
[LibreOfficeIOM.zip](https://github.com/haikuports/haikuports/files/2439349/LibreOfficeIOM.zip)

Well, finally I searched... anyway... firstimer issues. here is the LibreOfficeIOM icon, just unzip it and opens directly in IOM.
All these images and file uploads could not be done from Haiku... that is a drawback.

Thanks all of you.
Regards,
RR

**Comment by fbrosson** [comment]
Thanks @roired for the ZIP file.
I've just pushed the IOM file. I guess we can include it in this commit.
(I can revert this if needed.)

Shall I merge?

**Comment by scottmc** [comment]
Yes, merge.  Nice work on the icon.

**Pull request merged**